assets/src/scripts/components/SearchBar/SearchBarResult.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+/**
+ * History List.
+ */
+export default class SearchBar_Result extends React.Component {
+
+	render() {
+
+		var classes = [ 'SearchBar_Result' ];
```

Comment:
Is there any advantage to using `let` here? My understanding is it restricts the scope of the var to the current block, but since there are no other blocks in this function it makes no difference.

Again, if there are standards or other benefits to this, happy to change.